lib/ykpiv.c

```diff
@@ -154,8 +156,7 @@ ykpiv_rc ykpiv_connect(ykpiv_state *state, const char *wanted) {
     }
     rc = SCardConnect(state->context, reader_ptr, SCARD_SHARE_SHARED,
         SCARD_PROTOCOL_T1, &state->card, &active_protocol);
-    if(rc != SCARD_S_SUCCESS)
-    {
+    if(rc != SCARD_S_SUCCESS) {
```

Comment:
Yeah, I'm fine with the change itself, what I'm trying to do is keep the amount of different things in each PR to a minimum for easier review and merging.
In addition to this there is a bunch of if and else without {} at all that should be fixed, please drop this from this PR and make a new one for fixing these if you want (or I'll do that).